.travis.yml

```diff
@@ -21,6 +21,10 @@ before_script:
   - go get -u github.com/ChimeraCoder/gojson/gojson
   - go get -u github.com/golang/protobuf/protoc-gen-go
   - go get -u github.com/gogo/protobuf/protoc-gen-gofast
+  - pushd $GOPATH/src/github.com/gogo/protobuf/
+  - git checkout v0.5
+  - go install ./protoc-gen-gofast
```

Comment:
nit: I'd rather do `go install github.com/gogo/protobuf/protoc-gen-gofast`. Relative imports (even to the build tool) are not recommended and can technically break between versions :)